### ISSUE TITLE
NameError (NO_COREF_LIST) when importing en_coref_md

### ISSUE THREAD

**Issue by ZiningZhu** [huggingface/neuralcoref#72]
I was given a NameError when importing `en_coref_md`:
```python
import en_coref_md
nlp = en_coref_md.load()
```
After ~20s, got this error message: 
```
NameError: name 'NO_COREF_LIST' is not defined
Exception NameError: "name 'NO_COREF_LIST' is not defined" in 'en_coref_md.neuralcoref.neuralcoref.get_hash_lookups' ignored
```

I also tried the alternative method:
```python
import spacy
nlp = spacy.load('en_coref_md')
```
Running the `spacy.load` the first time gives `File "neuralcoref.pyx", line 101, in init en_coref_md.neuralcoref.neuralcoref TypeError: must be char, not unicode` (same as Issue #48), and the second time gives `NO_COREF_LIST` NameError.

Any ideas what can be done to avoid this?

Thanks!

Configuration: macOS Sierra 10.13.5, python 2.7.10 (GCC 4.2.1), spacy 2.0.11, inside a `virtualenv` environment. 
Note: Already did a `pip install --upgrade --force-reinstall --no-cache-dir MODEL_URL` where MODEL_URL is [en_coref_md](https://github.com/huggingface/neuralcoref-models/releases/download/en_coref_md-3.0.0/en_coref_md-3.0.0.tar.gz).

**Comment by ZiningZhu** [comment]
Also `neuralcoref` installed should be version 3.0.0:
```bash
>>> import en_coref_md
>>> en_coref_md.__version__
u'3.0.0'
```

**Comment by pradipcyb** [comment]
having the same issue

**Comment by tanzf** [comment]
having the same issue

**Comment by thomwolf** [comment]
These installation issues should be fixed in the new release (4.0) with SpaCy 2.1+ (which is also on PyPI now). See the README for more details.
Please open a new issue if you still experience problems.
